### PR TITLE
Add python wrapper for reduce_sum

### LINKED ISSUE #6702: Need add python wrapper for reduce_sum

### PR DESCRIPTION
**Pull request by guoshengCS**
resolve #6702

### CODE REVIEW

File: python/paddle/v2/fluid/layers/nn.py [hunk 1]
```diff
@@ -789,3 +789,51 @@ def conv2d_transpose(input,
...
+        Variable: The reduced Tensor variable.
+    
+    Examples:
+        .. code-block:: python
+
+            # x is a Tensor variable with following elements:
+            #    [[0.2, 0.3, 0.5, 0.9]
+            #     [0.1, 0.2, 0.6, 0.7]]
+            fluid.layers.reduce_sum(x)  # output tensor: [3.5]
+            fluid.layers.reduce_sum(x, dim=0)  # output tensor: [0.3, 0.5, 1.1, 1.6]
```
Comment (pkuyym) [inline python/paddle/v2/fluid/layers/nn.py]: Please make sure the number of words in a line less than 80.

File: python/paddle/v2/fluid/layers/nn.py [hunk 2]
```diff
@@ -789,3 +789,51 @@ def conv2d_transpose(input,
...
+            fluid.layers.reduce_sum(x, dim=-1)  # output tensor: [1.9, 1.6]
+            fluid.layers.reduce_sum(x, dim=1, keep_dim=True)  # output tensor: [[1.9], [1.6]]
```
Comment (pkuyym) [inline python/paddle/v2/fluid/layers/nn.py]: Same as above.

File: python/paddle/v2/fluid/layers/nn.py (hunk 1 above)
Comment (guoshengCS) [reply python/paddle/v2/fluid/layers/nn.py]: Done.

File: python/paddle/v2/fluid/layers/nn.py (hunk 2 above)
Comment (guoshengCS) [reply python/paddle/v2/fluid/layers/nn.py]: Done.

**Pull request merged**